### PR TITLE
0.9.9 Spanish Translation

### PR DESCRIPTION
**Pull request by banakito**
Simply a rdm_es_ES.ts resource file translated to spanish.

### CODE REVIEW

File: src/resources/translations/rdm_es_ES.ts [hunk 1]
```diff
@@ -0,0 +1,1218 @@
...
+    <name>RDM</name>
+    <message>
+        <location filename="../../app/app.cpp" line="53"/>
+        <source>Settings directory is not writable</source>
+        <translation>El directorio de ajustes no es grabable</translation>
+    </message>
+    <message>
+        <location filename="../../app/app.cpp" line="55"/>
+        <source>RDM can&apos;t save connections file to settings directory. Please change file permissions or restart RDM as administrator.</source>
+        <translation>RDM no puede grabar el fichero de conexiones en el directorio de ajustes. Por favorcambia los permisos de fichero o reinicia RDM como administrador.</translation>
```
Comment (artrz) [inline src/resources/translations/rdm_es_ES.ts]: favorcambia -> favor cambia

Comment (banakito) [reply src/resources/translations/rdm_es_ES.ts]: Updated.
Thank you, Arturo.

Comment (artrz) [reply src/resources/translations/rdm_es_ES.ts]: You're welcome

**Comment by uglide** [comment]
Great, thanks

**Pull request merged**